extractor/requirements.txt

```diff
@@ -6,4 +6,5 @@ flask_sqlalchemy==2.3.2
 pandas==0.25.3
 psycopg2-binary==2.7.7
 requests==2.21.0
-uWSGI==2.0.18
\ No newline at end of file
+uWSGI==2.0.18
+cx_Oracle==7.1.2
```

Comment:
Maybe it's a bit picky but can you put the requirements in alphabetical order? 😅

Reply:
We don't have sqlAlchemy in the reqs?